api/RadioNodeList.json

```diff
@@ -54,10 +54,10 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#dom-radionodelist-value",
           "support": {
             "chrome": {
-              "version_added": "34"
+              "version_added": "21"
```

Comment:
Looks like this is where this was all implemented:
https://trac.webkit.org/changeset/115009/webkit
https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=115009

WebKit 536.9 would suggest Chrome 20, but if tests say 21 I trust that.

Can you double check the Safari versions? I think they should both be 6.

Reply:
It appears that this wasn't present in Safari 6, but is in Safari 8.  I haven't confirmed Safari 7 support (my VMs aren't working properly at the moment), but I'll set it to "7" based upon the timing of the commit.

Reply:
7 sounds good!